python/tests/util/storage_test.py

```diff
@@ -137,7 +137,7 @@ def verify_library(ac):
 
 
 def is_strategy_branch_valid_format(input_string):
-    pattern = r"^(linux|windows)_cp3(6|7|8|9|10|11).*$"
+    pattern = r"^(linux|windows)_cp3(6|7|8|9|10|11|12|13).*$"
```

Comment:
No, we still want to install 3.6 in the persistent tests with the oldest version that we support for backwards compatibility.

I've also added 13 because we should add it soon, so better to save whoever is implementing it some headaches.